tests/testthat/test-osmdata.R

```diff
@@ -3,6 +3,8 @@
 # cache folder only used for testing purposes. This is achieved via the
 # [`temp_cache_dir()`] helper function, which should be called in each test.
 
+bb <- get_osm_bb("Bucharest")
+
```

Comment:
I would move this inside the only function that uses `bb` - it seems to be only one, and so we would also skip it if running in CI.